### ISSUE TITLE
TS can't find package types of `v5`

### ISSUE THREAD

**Issue by mikestopcontinues** [aleclarson/vite-tsconfig-paths#152]
After upgrading, typescript can no longer find the proper types.

```ts
import tsconfigPlugin from 'vite-tsconfig-paths';
```

```
Cannot find module 'vite-tsconfig-paths' or its corresponding type declarations.
  There are types at '/Users/mike/Code/@browsercat/core/tools/vite-config/node_modules/vite-tsconfig-paths/dist/index.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.ts(2307)
```

I can fix the error by changing "module" to "preserve", but that causes other issues in my repo.

I think this plugin should work with older esm-compatible tsconfig options. For example, my current config:

```json
{
  "compilerOptions": {
    "module": "es2022",
    "moduleResolution": "node",
  }
}
```

**Comment by aleclarson** [comment]
Sorry about that! Fixed in v5.0.1

**Comment by mikestopcontinues** [comment]
Thanks!
